sites/all/modules/reol_base/reol_base.install

```diff
@@ -347,3 +347,10 @@ function reol_base_update_7128() {
 function reol_base_update_7129() {
   module_enable(['reol_shared_theme'], TRUE);
 }
+
+/**
+ * Enable reol_field_carousel module.
```

Comment:
No need to enable it yet. We'll add this update hook when we'll actually need it in a node type.